Directory.Build.targets

```diff
@@ -12,28 +12,33 @@
   <ItemGroup>
     <!--3rd party dependencies-->
     <PackageReference Update="Microsoft.NET.Test.Sdk" Version="16.11.0" />
-	  <PackageReference Update="Moq" Version="4.16.1" />
-	  <PackageReference Update="NUnit" Version="3.13.2" />
-	  <PackageReference Update="NUnit3TestAdapter" Version="4.1.0" />
-    <PackageReference Update="Microsoft.Data.Sqlite" Version="6.0.0" />
+	<PackageReference Update="Moq" Version="4.16.1" />
+	<PackageReference Update="NUnit" Version="3.13.2" />
+	<PackageReference Update="NUnit3TestAdapter" Version="4.1.0" />
 
     <PackageReference Update="Newtonsoft.Json" Version="12.0.3" />
     <PackageReference Update="CommandLineParser" Version="2.8.0" />
-
-    <PackageReference Update="Microsoft.EntityFrameworkCore" Version="6.0.0" />
+	  
+	<FrameworkReference Update="Microsoft.AspNetCore.App" Version="6.0.0"/>
```

Comment:
does it make sense to have one varible for the net versions in order to know what belongs together?

Reply:
yeah sure 😅